### PR TITLE
feat: Add ability to use Fargate Ephemeral Storage

### PR DESCRIPTION
**Pull request by raizyr**
## Description
Add the ability to use Fargate Ephemeral Storage

## Motivation and Context
With multiple large plans being ran, it's possible to run the container out of disk space.  This enables the use of Fargate's ephemeral storage option to reduce that chance.

## Breaking Changes
None

## How Has This Been Tested?
- [ ] I have tested and validated these changes using one or more of the provided `examples/*` projects
Been running it successfully for a couple months.

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -625,3 +625,19 @@ variable "ecs_service_enable_execute_command" {
   type        = bool
   default     = true
 }
+
+variable "enable_ephemeral_storage" {
+  description = "Enable to use Fargate Ephermal Storage"
+  type        = bool
+  default     = false
+}
+variable "ephemeral_storage_size" {
```
[the remark this answers is not in the document]
Comment (raizyr) [reply variables.tf:641]: Done

**Comment by raizyr** [comment]
I missed that this had been reviewed for about a month, sorry for the delay.  Requested changes have been made. 😄

File: examples/github-complete/README.md [hunk 2]
```diff
@@ -30,22 +30,22 @@ Go to https://eu-west-1.console.aws.amazon.com/ecs/home?region=eu-west-1#/settin
...
 |------|---------|
-| <a name="provider_aws"></a> [aws](#provider\_aws) | >= 2.68 |
+| <a name="provider_aws"></a> [aws](#provider\_aws) | >= 3.45 |
 
 ## Modules
 
 | Name | Source | Version |
 |------|--------|---------|
-| <a name="module_atlantis"></a> [atlantis](#module\_atlantis) | ../../ |  |
+| <a name="module_atlantis"></a> [atlantis](#module\_atlantis) | ../../ | n/a |
```
Comment (antonbabenko) [inline examples/github-complete/README.md:46]: Looks good, but please use `terraform-docs` version 0.13 to fix the docs (I don't have write permissions to fix it for you).

Comment (raizyr) [reply examples/github-complete/README.md]: Done 👍🏻

**Comment by antonbabenko** [comment]
Thanks, @raizyr !

[v3.5.0](https://github.com/terraform-aws-modules/terraform-aws-atlantis/releases/v3.5.0) has been just released.

**Pull request merged**